exercises/I-array-properties/exercise.js

```diff
@@ -6,8 +6,13 @@
 */
 
 function isEmpty(arr) {
-  return; // complete this statement
+  if(arr.length){
```

Comment:
This would not work. The arr has a length even if it is empty. However, it will be equal to 0, so you should compare it to 0.